### ISSUE TITLE
Feature request: Category filter for PS1 Classics

### ISSUE THREAD

**Issue by 13xforever** [RPCS3/rpcs3#5378]
Right now they fall into Game category, but it would be neat to hide them from the normal PS3 titles.
In `param.sfo` PS1 classics have `CATEGORY` set to `1P` (as opposed to `HG` for digital PS3 games).

**Comment by nitrohigito** [comment]
This "normal PS3 titles" thing is an issue for the compatiblity database as well, and has been talked about in decent length over at #testers, and I think, maybe an application-oriented listing approach would be the best suited overall.

What I mean by that is properly tagging all applications, pull that from the wiki on-demand, and have a separate "Games" and "Other applications" lists maybe. This isn't to say that this approach laid out above shouldn't be put in or is bad - this generalizational approach could be a great "next step" in this matter on a later date.

TLDR: Agreed.

**Comment by Megamouse** [comment]
Should be resolved with #5382
